Calling/ClientApp/src/feedbacks/submitFeedback.ts

```diff
@@ -0,0 +1,37 @@
+import { BlobServiceClient } from '@azure/storage-blob';
+import { Feedback } from './Feedback';
+
+export const uploadFeedback = async (feedback: Feedback, screenShotCanvas: HTMLCanvasElement | undefined): Promise<void> => {
+  try {
+    const response = await fetch('/blobSettings');
+    if (!response.ok) {
+      throw new Error('Failed to get blob settings from server!');
+    }
+    const settings = await response.json();
+
+    const blobServiceClient = new BlobServiceClient(settings.sasUri);
+    const containerClient = blobServiceClient.getContainerClient(settings.containerName);
+    const blockBlobClient = containerClient.getBlockBlobClient('logs-' + feedback.guid + '.json');
```

Comment:
we can probably use a templated string here instead `logs-${feedback.guid}.json`